projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/close-account.core-e2e-spec.ts

```diff
@@ -69,6 +69,8 @@ describe('My Account - Close Account', () => {
         alerts
           .getSuccessAlert()
           .should('contain', 'Account closed with success');
+
+        cy.get('cx-login .cx-login-greet').should('not.exist');
```

Comment:
The login component should be existing in some form when a page has successfully loaded. This does not test that we have successfully logged out but only that the DOM is changing and the element has disappeared temporarily.

Instead, we should look for the button to sign in.

Reply:
This is checking for the greeting message where it states "Hi 'user'" when a user is logged in